SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.m

```diff
@@ -498,6 +406,44 @@ - (OWSSignalServiceProtosDataMessage *)buildDataMessage:(NSString *_Nullable)rec
     OWSSignalServiceProtosDataMessageBuilder *builder = [self dataMessageBuilder];
     [builder setTimestamp:self.timestamp];
     [builder addLocalProfileKeyIfNecessary:self.thread recipientId:recipientId];
+
+    if (self.quotedMessage) {
```

Comment:
IIRC we now correctly send protos with quoted replies.  I don't think I did the work to parse incoming protos with quoted replies.